substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -15,107 +15,7 @@
-//!
-//! ##### Display the `GenesisConfig` preset with given name
-//!
-//! Displays the content of the `GenesisConfig` preset for given name
-//! ```bash
-//! chain-spec-builder display-preset -r runtime.wasm -p "staging"
-//! ```
-//! 
-//! _Note:_ [`GenesisBuilder::get_preset`][sp-genesis-builder-get-preset] runtime function is called.
-//!
-//! ##### List the names of `GenesisConfig` presets provided by runtime.
-//!
-//! Displays the names of the presets of `GenesisConfigs` provided by runtime.
-//! ```bash
-//! chain-spec-builder list-presets -r runtime.wasm
-//! ```
-//! 
-//! _Note:_ [`GenesisBuilder::preset_names`][sp-genesis-builder-list] runtime function is called.
-//!
-//! ##### Generate chain spec using runtime provided genesis config preset.
-//!
-//! Patch the runtime's default genesis config with the named preset provided by the runtime and generate the plain
-//! version of chain spec:
-//! ```bash
-//! chain-spec-builder create -r runtime.wasm named-preset "staging"
-//! ```
-//! 
-//! _Note:_ [`GenesisBuilder::get_preset`][sp-genesis-builder-get-preset] and [`GenesisBuilder::build_state`][sp-genesis-builder-build] runtime functions are called.
-//!
-//! ##### Generate raw storage chain spec using genesis config patch.
-//!
-//! Patch the runtime's default genesis config with provided `patch.json` and generate raw
-//! storage (`-s`) version of chain spec:
-//! ```bash
-//! chain-spec-builder create -s -r runtime.wasm patch patch.json
-//! ```
-//! 
-//! _Note:_ [`GenesisBuilder::build_state`][sp-genesis-builder-build] runtime function is called.
-//!
-//! ##### Generate raw storage chain spec using full genesis config.
-//!
-//! Build the chain spec using provided full genesis config json file. No defaults will be used:
-//! ```bash
-//! chain-spec-builder create -s -r runtime.wasm full full-genesis-config.json
-//! ```
-//! 
-//! _Note_: [`GenesisBuilder::build_state`][sp-genesis-builder-build] runtime function is called.
-//!
-//! ##### Generate human readable chain spec using provided genesis config patch.
-//! ```bash
-//! chain-spec-builder create -r runtime.wasm patch patch.json
-//! ```
-//! 
-//! ##### Generate human readable chain spec using provided full genesis config.
-//! ```bash
-//! chain-spec-builder create -r runtime.wasm full full-genesis-config.json
-//! ```
-//! 
-//! ##### Extra tools.
-//! The `chain-spec-builder` provides also some extra utilities: [`VerifyCmd`], [`ConvertToRawCmd`],
-//! [`UpdateCodeCmd`].
-//!
-//! [`sc-chain-spec`]: ../sc_chain_spec/index.html
-//! [`node-cli`]: ../node_cli/index.html
-//! [`sp-genesis-builder`]: ../sp_genesis_builder/index.html
-//! [sp-genesis-builder-build]: ../sp_genesis_builder/trait.GenesisBuilder.html#method.build_state
-//! [sp-genesis-builder-list]: ../sp_genesis_builder/trait.GenesisBuilder.html#method.preset_names
-//! [sp-genesis-builder-get-preset]: ../sp_genesis_builder/trait.GenesisBuilder.html#method.get_preset
+#![doc = include_str!("../README.md")]
```

Comment:
From what see in `docify crate` [here](https://github.com/sam0x17/docify/blob/206e6867b6405202a9f1913ccb5ae0c15d4a3d30/src/lib.rs#L79-L80), we don't need extra wrapping function for this.
```suggestion
#![doc = include_str!("../README.md")]
#[cfg(feature = "generate-readme")]
docify::compile_markdown!("README.docify.md", "README.md");
}

```

Reply:
Strange are the macro ways. Will try, thanks for trying it. 🙏

Reply:
[6f9c200](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/6f9c200325c3480b2aab5b69507ea64d1bdf8e14)